src/components/Layout/Sidebar.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Avatar from './Avatar';
+import { UserProfile } from 'services/user';
+
+import styles from './Layout.module.css';
+
+interface ComponentProps {
+  show: boolean;
+  setShow: React.Dispatch<React.SetStateAction<boolean>>;
+  user: UserProfile;
+}
+
+const Sidebar = ({ show, setShow, user }: ComponentProps) => {
+  return (
+    <>
+      {show && (
+        <div
+          className="fixed top-0 right-0 h-screen w-screen"
+          data-testid="sidebar-overlay"
+          onClick={() => setShow(false)}
+        />
+      )}
+      <aside className={styles.sidebar} aria-hidden={!show}>
+        <section className="flex justify-between items-center">
```

Comment:
Do you plan to add the divider later when working on the logout feature?
![Screen Shot 2565-05-11 at 11 41 19](https://user-images.githubusercontent.com/1772999/167770662-31b18be1-6f42-4adf-b655-8f2b95c74107.png)

Reply:
Yes, I plan to add the divider in the logout feature. The divider without a logout link looks odd to me 😂